svg-template.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
![insurance_template](https://github.com/user-attachments/assets/efe6ce64-5e9c-4ead-83b7-7765e19d5209)
![national_id_template_without_qr_code](https://github.com/user-attachments/assets/b3e9bcde-cc0f-4eb9-9f58-a9540a021240)

Svg Templates for Insurance and Mosip VC needs to be updated.

Reply:
@Piyush7034 @vharsh

Reply:
Thanks Bala.